fcrepo4/fcrepo4
# Purge resources and fix FedoraLdpTests

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3034

# What does this Pull Request do?
Allows you to DELETE a `fcr:tombstone` to remove the sidecar files (if part of an Archival Group) or delete the entire object.

Also re-enables the FedoraLdpTests unit test class and fixes all the tests.

# How should this be tested?

Build the PR.
Create an object.
Find the object in your storage root.
Delete the object, see that the object still exists but the sidecar file has the `deleted: true` property.
Delete the tombstone, see the entire OCFL object is removed.

# Interested parties
@fcrepo4/committers
